skipchain/skipchain.go

```diff
@@ -68,6 +69,10 @@ type Service struct {
 // added.
 func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, onet.ClientError) {
 	prop := psbd.NewBlock
+	if !s.ServerIdentity().Equal(prop.Roster.Get(0)) {
```

Comment:
Does the verification function checks the ordering of the indivudual keys in the roster of a block ?